stories/Customize/CSSVariables.stories.mdx

```diff
@@ -172,7 +185,9 @@ a {
   </div>
 </Story>
 
-#### Dark theme [Experimental]
+<br/>
+
+### Dark theme
 
 <Story name="Dark theme">
```

Comment:
@nitinprojects _a Rename this story to "Dark mode".

Reply:
@praveen-murali-ind  can you update the name in CodeSandbox https://codesandbox.io/s/dark-theme-2dhsl0 and share the new code?

Reply:
@praveen-murali-ind _a